src/placement-center/src/lib.rs

```diff
@@ -120,8 +109,8 @@ impl PlacementCenter {
 
     pub async fn start(&mut self, stop_send: broadcast::Sender<bool>) {
         self.init_cache();
-        let (raft_message_send, raft_message_recv) = mpsc::channel::<RaftMessage>(1000);
-        let (peer_message_send, peer_message_recv) = mpsc::channel::<PeerMessage>(1000);
+        // TODO 这个通道的作用什么？  在删除raftV1 相关代码后，接收端没有使用的地方，但是发送端好多地方使用
```

Comment:
Chinese comments need to be removed @MangerLe

Reply:
> Chinese comments need to be removed @MangerLe

This information has been deleted